rpm/pacemaker.spec.in

```diff
@@ -308,8 +310,8 @@ when related resources fail and can be configured to periodically check
 resource health.
 
 Available rpmbuild rebuild options:
-  --with(out) : cibsecrets coverage doc stonithd hardening pre_release
-                profiling upstart_job
+  --with(out) : cibsecrets coverage doc hardening pre_release profiling stonithd
+                upstart_job
 
```

Comment:
do we actually still support stonithd from clusterglue?

Reply:
The option isn't actually for stonithd support, but support for stonithd agents (aka LHA, external/...). We should change the option name at some point, and if I had noticed it before rc1 I would have :)

Reply:
Aah didn't check the current behavior honestly just thought I remembered that back when Andrew had introduced rhcs-agent support it came together with tossing the cluster-glue stonith-daemon.